libvast/src/command.cpp

```diff
@@ -211,17 +211,45 @@ void doctext(const command& cmd, std::ostream& out) {
   out << std::left << cmd.documentation;
 }
 
+void manheader(std::ostream& out) {
+  out << "% VAST(1)\n"
+         "% Tenzir GmbH\n"
+         "\n"
+         "# NAME\n"
+         "\n"
+         "`vast` -- manage a VAST node\n"
+         "\n"
+         "# OVERVIEW\n"
+         "\n"
+         "This section describes the VAST system and its components from a "
+         "user "
+         "interaction point of view.\n"
```

Comment:
@dominiklohmann can you reformat this again? Looks like `clang-format` wasn't doing the job right.